src/include/duckdb/storage/serialization/logical_operator.json

```diff
@@ -165,7 +165,7 @@
       },
       {
         "id": 201,
-        "name": "projections",
```

Comment:
Can we avoid changing the name in the serialization? Otherwise the JSON serialization is not backwards compatible, can we do e.g. `"property": "projection_map"` instead of renaming the serialized member?